src/patternfly/components/Drawer/drawer.scss

```diff
@@ -1,112 +1,241 @@
+  --pf-c-drawer--child--m-padding--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-drawer--child--m-padding--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-drawer--child--m-padding--PaddingLeft: var(--pf-global--spacer--md);
+
+  // Responsive, modified padding, child padding
+  --pf-c-drawer--child--m-padding--md--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-drawer--child--m-padding--md--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-drawer--child--m-padding--md--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-drawer--child--m-padding--md--PaddingLeft: var(--pf-global--spacer--lg);
 
   @media screen and (min-width: $pf-global--breakpoint--md) {
     --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--md--FlexBasis);
-  }
 
-  // Panel body
-  --pf-c-drawer__panel-body--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-drawer__panel-body--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-drawer__panel-body--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-drawer__panel-body--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-drawer__panel-body--md--PaddingTop: var(--pf-global--spacer--lg);
-  --pf-c-drawer__panel-body--md--PaddingRight: var(--pf-global--spacer--lg);
-  --pf-c-drawer__panel-body--md--PaddingBottom: var(--pf-global--spacer--lg);
-  --pf-c-drawer__panel-body--md--PaddingLeft: var(--pf-global--spacer--lg);
+    // Responsive child padding
+    --pf-c-drawer--child--PaddingTop: var(--pf-c-drawer--child--md--PaddingTop);
+    --pf-c-drawer--child--PaddingRight: var(--pf-c-drawer--child--md--PaddingRight);
+    --pf-c-drawer--child--PaddingBottom: var(--pf-c-drawer--child--md--PaddingBottom);
+    --pf-c-drawer--child--PaddingLeft: var(--pf-c-drawer--child--md--PaddingLeft);
+
+    // Responsive, modified padding, child padding
+    --pf-c-drawer--child--m-padding--PaddingTop: var(--pf-c-drawer--child--m-padding--md--PaddingTop);
+    --pf-c-drawer--child--m-padding--PaddingRight: var(--pf-c-drawer--child--m-padding--md--PaddingRight);
+    --pf-c-drawer--child--m-padding--PaddingBottom: var(--pf-c-drawer--child--m-padding--md--PaddingBottom);
+    --pf-c-drawer--child--m-padding--PaddingLeft: var(--pf-c-drawer--child--m-padding--md--PaddingLeft);
+  }
 
-  @media screen and (min-width: $pf-global--breakpoint--md) {
-    --pf-c-drawer__panel-body--PaddingTop: var(--pf-c-drawer__panel-body--md--PaddingTop);
-    --pf-c-drawer__panel-body--PaddingRight: var(--pf-c-drawer__panel-body--md--PaddingRight);
-    --pf-c-drawer__panel-body--PaddingBottom: var(--pf-c-drawer__panel-body--md--PaddingBottom);
-    --pf-c-drawer__panel-body--PaddingLeft: var(--pf-c-drawer__panel-body--md--PaddingLeft);
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    --pf-c-drawer__panel--FlexBasis: var(--pf-c-drawer__panel--xl--FlexBasis);
+    --pf-c-drawer__panel--MinWidth: var(--pf-c-drawer__panel--xl--MinWidth);
```

Comment:
Should this var be defined outside of this media query? It's used in a declaration but undefined below this breakpoint.